api/src/pcapi/core/subscription/ubble/api.py

```diff
@@ -134,8 +135,14 @@ def handle_validation_errors(
 ) -> None:
     error_codes = fraud_check.reasonCodes or []
     relevant_error_code = get_most_relevant_ubble_error(error_codes)
-    if relevant_error_code in ubble_fraud_constants.REASON_CODE_REQUIRING_EMAIL_UPDATE:
-        transactional_mails.send_subscription_document_error_email(user.email, relevant_error_code)
+    if (
+        relevant_error_code in ubble_fraud_constants.REASON_CODE_REQUIRING_IMMEDIATE_NOTIFICATION_REMINDER
+        or relevant_error_code in ubble_fraud_constants.REASON_CODE_REQUIRING_IMMEDIATE_EMAIL_REMINDER
+    ):
+        if relevant_error_code in ubble_fraud_constants.REASON_CODE_REQUIRING_IMMEDIATE_EMAIL_REMINDER:
+            transactional_mails.send_subscription_document_error_email(user.email, relevant_error_code)
+        if relevant_error_code in ubble_fraud_constants.REASON_CODE_REQUIRING_IMMEDIATE_NOTIFICATION_REMINDER:
+            track_ubble_ko_event(user.id, relevant_error_code)
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Je comprend l'idée mais c'est dur à lire. Le premier `if` n'est pas utile
Le code suivant fait la même chose:
```python
if relevant_error_code in ubble_fraud_constants.REASON_CODE_REQUIRING_IMMEDIATE_NOTIFICATION_REMINDER:
    track_ubble_ko_event(user.id, relevant_error_code)
if relevant_error_code in ubble_fraud_constants.REASON_CODE_REQUIRING_IMMEDIATE_EMAIL_REMINDER:
    transactional_mails.send_subscription_document_error_email(user.email, relevant_error_code)

Reply:
En effet, je n'étais moi-même pas satisfait en l'écrivant, mais je n'avais pas mieux sans faire d'autres refacto.
En pratique oui ça fait la même chose, parce que `REASON_CODE_REQUIRING_IMMEDIATE_NOTIFICATION_REMINDER` est inclue dans `REASON_CODE_REQUIRING_IMMEDIATE_EMAIL_REMINDER` dans l'état actuel des choses.
En théorie non, vu que tu perds l'exclusivité du `if-else`.
Tu penses que c'est mieux de supprimer le premier `if` ?

Reply:
Dans ce cas :

```python
if relevant_error_code in ubble_fraud_constants.REASON_CODE_REQUIRING_IMMEDIATE_NOTIFICATION_REMINDER: 
    track_ubble_ko_event(user.id, relevant_error_code) 
elif relevant_error_code in ubble_fraud_constants.REASON_CODE_REQUIRING_IMMEDIATE_EMAIL_REMINDER: 
    transactional_mails.send_subscription_document_error_email(user.email, relevant_error_code)
else:
    # la suite

Reply:
Dans ce cas on perd la possibilité d'envoyer le mail et la notification.

Reply:
Oui c'est vrai
Extrait le dans une fonction ? 
Comme ça le mon aide à lire

Reply:
Merci pour la review !